ui/app/components/variable-paths.hbs

```diff
@@ -31,6 +31,7 @@
           <LinkTo
             @route="variables.variable"
             @model={{file.absoluteFilePath}}
+            @query={{hash namespace="*"}}
```

Comment:
Since we're in the same route (and the parent route) is still activated, `LinkTo` will automatically forward the current query parameter. To override this behavior, we need to explicitly reset this parameter back to its original value `*`.